biz.aQute.bnd.gradle/test/aQute/bnd/gradle/TestBndPlugin.groovy

```diff
@@ -123,7 +123,7 @@ class TestBndPlugin extends Specification {
           bndrun.isFile()
           props.load(bndrun, new Slf4jReporter(TestBndPlugin.class))
           props.getProperty('-runbundles') =~ /osgi\.enroute\.junit\.wrapper/
-          result.output =~ Pattern.quote('Unresolved requirements: [test.simple]')
+          result.output =~ '(?s)Unresolved requirements:(.*)\\s+test.simple\\n'
```

Comment:
pretty sure it's the explicit newline char at the end. I need to change it to `\\s+` because it's already in newline mode.